Test/AccountTest.cs

```diff
@@ -146,5 +147,19 @@ public void GetAccountNotes()
 
             notes.Should().BeEmpty();
         }
+
+        [RecurlyFact(TestEnvironment.Type.Integration)]
+        public void Balance()
```

Comment:
@bhelx With this test I get back a balance, but the API is not sending back the `PastDue`. How can I modify this test so that I can test the population of that property as well?